src/definition/ui/IUIActionButtonDescriptor.ts

```diff
@@ -30,8 +30,9 @@ export interface IUActionButtonWhen {
 
 export interface IUIActionButtonDescriptor {
     actionId: string;
-    labelI18n: string;
     context: UIActionButtonContext;
+    labelI18n: string;
+    variant?: 'danger';
```

Comment:
Just clarifying, this means that the `variant` prop can either be omitted in the descriptor or be set to `'danger'`, is that correct? I'm thinking if it wouldn't be worth it to add a `'normal'` variant - it feels more ergonomic for situations where I need to dynamically assign the danger vairant, where I can do like `variant: isDanger ? 'danger' : 'normal'`, instead of `...isDanger && { variant: 'danger' }`